hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/CreateVolumeHandler.java

```diff
@@ -59,13 +61,19 @@ protected void execute(OzoneClient client, OzoneAddress address)
     VolumeArgs.Builder volumeArgsBuilder = VolumeArgs.newBuilder()
         .setAdmin(adminName)
         .setOwner(ownerName);
-    if (quotaOptions.getQuotaInBytes() != null) {
-      volumeArgsBuilder.setQuotaInBytes(OzoneQuota.parseQuota(
-          quotaOptions.getQuotaInBytes(),
-          quotaOptions.getQuotaInNamespace()).getQuotaInBytes());
+    if (!Strings.isNullOrEmpty(quotaOptions.getQuotaInBytes())) {
+      volumeArgsBuilder.setQuotaInBytes(OzoneQuota.parseSpaceQuota(
+          quotaOptions.getQuotaInBytes()).getQuotaInBytes());
+    } else {
+      volumeArgsBuilder.setQuotaInBytes(OzoneConsts.QUOTA_RESET);
```

Comment:
With default quotaInBytes and quotaInNamespace value, do we still need this and below else statement?

Reply:
This part is superfluous and can be deleted